demo/src/main/java/com/google/android/fhir/demo/SyncFragment.kt

```diff
@@ -72,25 +76,42 @@ class SyncFragment : Fragment() {
     }
   }
 
-  private fun currentSyncJobStatus(currentSyncJobStatus: CurrentSyncJobStatus) {
-    requireView().findViewById<TextView>(R.id.current_status).text =
+  private fun collectSyncJobStatus(currentSyncJobStatus: CurrentSyncJobStatus) {
```

Comment:
Nit: Earlier name of the function was better
```suggestion
  private fun currentSyncJobStatus(currentSyncJobStatus: CurrentSyncJobStatus) {
```

Reply:
done.